json-schema/src/main/kotlin/io/bkbn/kompendium/json/schema/definition/JsonSchema.kt

```diff
@@ -27,6 +27,7 @@ sealed interface JsonSchema {
         is NullableDefinition -> NullableDefinition.serializer().serialize(encoder, value)
         is OneOfDefinition -> OneOfDefinition.serializer().serialize(encoder, value)
         is AnyOfDefinition -> AnyOfDefinition.serializer().serialize(encoder, value)
+        else -> error("Unknown JsonSchema type: ${value::class.simpleName}")
```

Comment:
`JsonSchema` should be sealed so I think we don't need this `else`

Reply:
Yup, IntelliJ was bugging. Will close PR and make a new one anyway :)